lab09/lab_09.cpp

```diff
@@ -10,13 +10,25 @@
 using namespace std;
 
 
+// Fill the arrays with the amount of data in the input file
 void fill_arrays(string patron_name[], char book_type[], double days_overdue[], int &amount_of_records);
+
+// Fill the fine_cost array with the amount of fines owed per patron
 void calculate_fines(double days_overdue[], int amount_of_records, double fine_cost[], char book_type[]);
+
+// Sort the arrays from the highest fine cost to the lowest fine cost
 void sort_high_to_low(double days_overdue[], double fine_cost[], char book_type[], string patron_name[], int amount_of_records);
+
+// Send the fine sorted arrays to file output
 void send_to_output_high_to_low(double fine_cost[], string patron_name[], int amount_of_records);
+
+// Sort the arrays by the alphabetical order of the patrons names
 void sort_alphabetically(double days_overdue[], double fine_cost[], char book_type[], string patron_name[], int amount_of_records);
+
+// Send the alphabetically sorted arrays to file output
 void send_to_output_alphabetically(double days_overdue[], double fine_cost[], char book_type[], string patron_name[], int amount_of_records);

```

Comment:
This stuff is the precursor to using `.h` files.  So you actually want to put some high level documentation on these defs.

Reply:
All your function definitions should use consistent argument order.  E.g., everything should be:

```cpp
void function(string patron_name[], char book_type[], double days_overdue[], additional stuff here....)
```

Your definitions move things around, sometimes patron_name is before book_type, sometimes after.

Consistency is your friend.  Once you have the order in your head (stack), you don't have to re-learn a new order for another function.

You can put your output arrays (the things you copying into), at the beginning of the args list, or at the end.  But just be consistent.